pkg/machine/ignition.go

```diff
@@ -239,6 +273,21 @@ func getFiles(usrName string) []File {
 		},
 	})
 
+	files = append(files, File{
+		Node: Node{
+			Path: "/etc/tmpfiles.d/podman-docker.conf",
+		},
+		FileEmbedded1: FileEmbedded1{
+			Append: nil,
+			// Create a symlink from the docker socket to the podman socket.
+			// Taken from https://github.com/containers/podman/blob/main/contrib/systemd/system/podman-docker.conf
+			Contents: Resource{
+				Source: strToPtr("data:,L%2B%20%20%2Frun%2Fdocker.sock%20%20%20-%20%20%20%20-%20%20%20%20-%20%20%20%20%20-%20%20%20%2Frun%2Fpodman%2Fpodman.sock"),
```

Comment:
Knowing nothing of ignition, are these control chars necessary %2 and %20?

Reply:
Yes, they are necessary but we should really start calling a function such that the values are human readable.

Reply:
I'll introduce one in my follow-up since I already have other changes to make.

Reply:
Done, and converted everything else. Readability is quite a bit better now.